lib/seed/Seeder.js

```diff
@@ -41,22 +44,25 @@ class Seeder {
   async _listAll(config) {
     this.config = this.setConfig(config);
     const loadExtensions = this.config.loadExtensions;
-    return readdir(this._absoluteConfigDir()).then((seeds) =>
-      seeds
-        .filter((value) => {
-          const extension = path.extname(value);
-          return includes(loadExtensions, extension);
-        })
-        .sort()
-    );
+    const seeds = (
+      await Promise.all(
+        this._absoluteConfigDirs().map(getFilepathsInFolderRecursively)
+      )
+    ).flat();
+    return seeds
+      .filter((value) => {
+        const extension = path.extname(value);
+        return includes(loadExtensions, extension);
+      })
+      .sort();
   }
 
   // Ensures a folder for the seeds exist, dependent on the
   // seed config settings.
   async _ensureFolder() {
-    const dir = this._absoluteConfigDir();
-
-    await ensureDirectoryExists(dir);
+    const dirs = this._absoluteConfigDirs();
+    const promises = dirs.map(ensureDirectoryExists);
+    return Promise.all(promises);
```

Comment:
If you want to be consistent with original implementation, should be `await Promise.all(promises);`.
Alternatively function doesn't need to be async anymore.

Reply:
I choose the second option, to match the `MigrationGenerator` implementation.
I don't know where to put this "common" method.
Do we have to make a `class` and make `MigrationGenerator` and `Seeder` ones extends them ?

Reply:
Either way works, you can have a superclass, or just import from some kind of commons util file with static functions.